examples/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ set(EXAMPLES_LIST
     custom-stopping-criterion
     ginkgo-overhead
     minimal-cuda-solver
+    mixed-spmv
```

Comment:
I think it should be put above (in the `EXAMPLES_EXEC_LIST`), since it uses an executor.

Reply:
I think `EXAMPLES_EXEC_LIST` is for validating the examples